frontend/packages/yki/src/redux/reducers/publicIdentification.ts

```diff
@@ -30,6 +30,9 @@ const publicIdentificationSlice = createSlice({
     rejectEmailLinkOrder(state) {
       state.emailLinkOrder.status = APIResponseStatus.Error;
     },
+    resetPublicIdentificationState() {
```

Comment:
Tätä ei vielä käytetä missään. Pitäis kai lisätä Done-steppiin sekä rekisteröinnin keskeyttämisen yhteyteen.